medusa/providers/torrent/json/t411.py

```diff
@@ -32,10 +33,25 @@
     try_int,
 )
 from medusa.providers.torrent.torrent_provider import TorrentProvider
+from medusa.scene_exceptions import get_scene_exceptions
 
 from requests.auth import AuthBase
 from requests.compat import quote, urljoin
 
+EPISODE_MAP = {
```

Comment:
Please add some comments with information about what this episode and season map really does. You might know. But someone touching this code in 1 year, will really not.